backend/cmm_helpers.ml

```diff
@@ -3882,6 +3882,8 @@ let infix_field_address ~dbg ptr n =
 
 let cint i = Cmm.Cint i
 
+let cint32 i = Cmm.Cint32 (Nativeint.of_int32 i)
```

Comment:
#2712 introduces a different `cint32` function. I think this one is better, and #2712 should be changed to use it.

Reply:
I'll do this